Modelplace_Cloud_API/Object_detection/README.md

```diff
@@ -118,6 +122,10 @@ If the program runs successfully, it should have a prediction folder that contai
 
 You can see an example of the content of these files on the [Faster R-CNN] by running "test model" using the [test image]. 
 
+<p align="center">
+    <img src="https://drive.google.com/uc?export=view&id=1oRsv1BII2SRgN6Rl8pq7sqdwAqb_FIz8" height="300"/>
+</p>
+
 ## Step 6: Drawing the results on your machine
```

Comment:
Drawing on the local machine requires the installation of an additional package modelplace-api. Then would it be better to put the drawing in a separate file? And prescribe how to run this file.